README.md

```diff
@@ -105,6 +105,13 @@ It is possible to interrupt the watcher from another thread with:
 <br>
 `watcher.finish` <br>
 
+8. Get entities filtered by labels <br>
```

Comment:
I think this section is great, I think it suits better to be part of item 1 in the readme where get_all_X is explained, rather then being number 8. it will increase the chances of clarity and it's more relevant there as it's an additional option for GET rather than a separate action.